### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by gleisonsilva** [Workiva/eva#120]
Hello! I'm searching for ways to use Datalog kind of queries in Clojure and come to here. Is this project alive? Can be considered an option to use for serious work 'today'?

**Comment by Saityi** [comment]
Hiya! Eva was used for serious work in production at the time it was open-sourced, but it is no longer under active development, and it may be difficult to get support for serious work today. There are some other options under active development that may suit your needs that you could check out:

- Datomic: https://www.datomic.com/
- Crux: https://opencrux.com/
- Datascript: https://github.com/tonsky/datascript

**Comment by tonyhb** [comment]
What did Workiva transition to, in transparency, and why?

**Comment by fmjrey** [comment]
> What did Workiva transition to, in transparency, and why?

Yes would be curious to know as well, though @Saityi did not really say they stopped using it, only that they stopped developing it, so I imagine it could still be used in some legacy estate?

**Comment by jesse-c** [comment]
Another alternative: [Datahike](https://github.com/replikativ/datahike)
